MOM_input

```diff
@@ -1,7 +1,7 @@
 ! This input file provides the adjustable run-time parameters for version 6 of the Modular Ocean Model (MOM6).
 
 ! === module MOM ===
-DT = 1800.0
+DT = 1350.0
```

Comment:
Should `DT_THERM` be an integer multiple of `DT`?

Reply:
Ideally, `DT_THERM` should be an integer multiple of `DT` and less than the forcing or coupling time-step (DT = 1350.0). However, there are no warnings or errors with the current `DT_THERM` but we have to reduce it, any thoughts @aekiss ?

Reply:
This issue will be handled in a separate PR as discussed [here](https://github.com/COSIMA/MOM6-CICE6/pull/48#issuecomment-2038431642)

Reply:
I think we need to decide what to do about `DT_THERM` here. The current settings are confusing:
`DT_THERM > DT` but `THERMO_SPANS_COUPLING = False (by default)`.

With these settings, I think **the value of `DT_THERM`  in the `MOM_input` will not be used** - instead `DT_THERM` will be reset internally to `DT`. If we do want to have `DT_THERM > DT` we need to set `THERMO_SPANS_COUPLING = True` which will require we also change to `DIABATIC_FIRST = False` due to limitations in the MOM6 NUOPC cap.

Thoughts @aekiss

Reply:
Options are:

1. Change to `DT_THERM = 1350.0` (the coupling timestep) and leave other setting as they are.
   
2. Leave `DT_THERM` as it is (`3600.0`) and change to `THERMO_SPANS_COUPLING = True` and `DIABATIC_FIRST = False`

Reply:
It's certainly confusing. Do we set `single_step_call` anywhere? It defaults to true, in which case `DT`, `DT_THERM` and `THERMO_SPANS_COUPLING` are ignored, [according to this](https://github.com/NCAR/MOM6/blob/13fbeb7/config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90#L168-L178).

Additional relevant documentation in MOM6 cap is [here](https://github.com/NCAR/MOM6/blob/13fbeb7/config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90#L293-L318).

Reply:
As discussed in today's meeting, let's do [option 1](https://github.com/COSIMA/MOM6-CICE6/pull/48#discussion_r1555126419) for now.

We can then explore option 2 with different `DT_THERM` in subsequent tests and PRs. (Note that when `THERMO_SPANS_COUPLING = True`, `DT_THERM` [will be rounded down to an integer number of coupling timesteps](https://github.com/NCAR/MOM6/blob/13fbeb7/config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90#L309-L314).)

Reply:
Given how confusingly these parameters interact, it will be good to have `MOM_parameter_doc.*` added to the repos to provide a sanity check on what is actually being used for the test when we do option 2 - see https://github.com/COSIMA/access-om3/issues/117. We could do this manually until CI is in place to automate it.

Reply:
If `DT_THERM` exceeds the coupling timestep, but the coupling timestep matches the dynamic timestep in CICE and baroclinic `DT` in MOM6, would it make sense to set the thermo timestep in CICE to match that in MOM?

This would involve setting `ndtd` in CICE to `DT_THERM/DT` and setting the CICE thermo timestep to `ndtd*DT`.

Reply:
Open question: can CICE run with a thermo timestep greater than the coupling timestep?